test/DistillTest.hs

```diff
@@ -0,0 +1,25 @@
+module DistillTest (distill1Test) where
+
+import Helpers
+import Test.HUnit
+import Term
+import Trans (dist)
+
+distill1Test :: IO Test
+distill1Test = let
+  c = "import Bool\n" ++

```

Comment:
Can we read both original and expected programs from files? I guess we should have a folder "gold" with expected programs.